_installation/single-multiple.md

```diff
@@ -26,16 +33,59 @@ CREATE USER babelfish_user WITH CREATEDB
 	CREATEROLE PASSWORD '<PUT_SECRET_PASS_HERE>' INHERIT;
 DROP DATABASE IF EXISTS demo;
 
+/* the Babelfish database */
 CREATE DATABASE demo OWNER babelfish_user;
+ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
+SELECT pg_reload_conf();
+
+/* determine the migration mode */
+ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 
 \c demo
 
 CREATE EXTENSION IF NOT EXISTS "babelfishpg_tds" CASCADE;
-ALTER SYSTEM SET babelfishpg_tsql.database_name = 'demo';
-ALTER DATABASE demo SET babelfishpg_tsql.migration_mode = 'single-db'|'multi-db';
 CALL SYS.INITIALIZE_BABELFISH('babelfish_user');
 ```
 
-The `ALTER DATABASE` statement assigns the correct parameter to the database.
-The init function will then perform all the steps needed to perform the setup.
+The `ALTER DATABASE` statement assigns the correct migration mode to the
+database.  The `SYS.INITIALIZE_BABELFISH` function will then perform all
+the steps needed to set up Babelfish.
+
+Once you are connected to the database via TDS, you can utilize the `USE`
```

Comment:
technically, this sentence is not correct. Better: "When you connect to the TDS port, the session connected to the PG database defined by babelfishpg_tsql.database_name. Inside this database, you can use the CREATE DATABASE and USE statements as you would in MSFT SQL Server."

Reply:
Hm, yes, that could be rewritten. I adapted your suggestion and ended up with
"When you connect to the TDS port, you will be connected to the PostgreSQL
database defined by `babelfishpg_tsql.database_name`.  You can then use the
`CREATE DATABASE` and `USE` statements as you would in Microsoft SQL Server.
(If you use `single-db` mode, you can only have a single T-SQL database.)"